frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { searchIndicator, SearchFormState } from './searchActions';
+import { Button, InputField } from 'govuk-react';
+import { spacing } from '@govuk-react/lib';
+import { useActionState } from 'react';
+import styled from 'styled-components';
+
+const StyledInputField = styled(InputField)(
+  spacing.withWhiteSpace({ marginBottom: 6 })
+);
+
+export const SearchForm = ({ indicator }: { indicator: string }) => {
+  const initialState: SearchFormState = {
+    indicator: '',
+    message: null,
+    errors: {},
+  };
+  const [state, formAction] = useActionState(searchIndicator, initialState);
+
+  console.log(`state ${JSON.stringify(state)}`);
```

Comment:
This probably doesn't need to be here, does it? In fact, what's the `useActionState` actually doing?

Reply:
Don't need the log. Will remove 👍 

useActionState simplifies the process of capturing form values and passing them to a server action as FormData.